tutorials/using-cloud-vpn-with-alibaba-with-redundancy

```diff
@@ -0,0 +1,950 @@
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+### Permissions
+
+To create a GCP network, a subnetwork, and other entities described in this
+guide, you must be able to sign in to GCP as a user who has
+[Network Admin](https://cloud.google.com/compute/docs/access/iam#network_admin_role)
+permissions. For details, see
+[Required Permissions](https://cloud.google.com/vpn/docs/how-to/creating-vpn-dynamic-routes#required_permissions)
+in the article
+[Creating a VPN Tunnel using Dynamic Routing](https://cloud.google.com/vpn/docs/how-to/creating-vpn-dynamic-routes).
+
+### IP Ranges
+
+The IP address ranges of the GCP VPC and the Alibaba VPC must not overlap.
+
+### GCP-compatible settings for IPSec and IKE
+
+Configuring the vendor side of the VPN network requires you to use IPsec and IKE
+settings that are compatible with the GCP side of the network. The following
+table lists settings and information about values compatible with GCP VPN.
+Use these settings for the procedures in the subsections that follow.
+
+<table>
+<thead>
+<tr>
+<th><strong>Setting</strong></th>
+<th><strong>Description or value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>IPsec Mode</td>
+<td>ESP+Auth Tunnel mode (Site-to-Site)</td>
+</tr>
+<tr>
+<td>Auth Protocol</td>
+<td>Pre-shared Key (psk)</td>
+</tr>
+<tr>
+<td>Shared Secret</td>
+<td>Also known as an IKE pre-shared key. Choose a strong password by following
+<a
+href="https://cloud.google.com/vpn/docs/how-to/generating-pre-shared-key">these
+guidelines</a>. The shared secret is very sensitive as it allows access
+into your network.</td>
+</tr>
+<tr>
+<td>Start</td>
+<td><code>Auto</code> (on-premises device should automatically restart the
+connection if it drops)</td>
+</tr>
+<tr>
+<td>PFS (Perfect Forward Secrecy)</td>
+<td>group1, group2, group5, group14, group24</td>
+</tr>
+<tr>
+<td>IKE ciphers</td>
+<td>aes, aes192, aes256, des, 3des <br>(For details about IKE ciphers for IKEv1 or IKEv2 supported by GCP,
+including the additional ciphers for PFS, see <a
+href="https://cloud.google.com/vpn/docs/concepts/supported-ike-ciphers">Supported
+IKE Ciphers</a>).</td>
+</tr>
+</tbody>
+</table>
+
+#### Recommended Alibaba Cloud VPN Gateway IKE proposal and policy
+Below are fields you may be asked to complete for the Alibaba Cloud side configurations. 
+The defaults indicated below will work with the defaults used on the GCP side.
+
+-  **Encryption algorithm**—<code>aes</code> 
+-  **Integrity algorithm**—<code>sha1</code>
+-  **Diffie-Hellman group**—<code>group2</code>
+-  **Lifetime/SA Life Cycels**—<code>86400</code> seconds 
+
+
+## Configuration Overview
+
+The Google Cloud VPN with Alibaba Cloud VPN Gateway configuration is comprised of the following steps.
+
+1. Configure the GCP side
+1. Configure the Alibaba Cloud side
+
+## Configure the GCP side
+
+This section covers the steps for creating a redundant GCP IPsec VPN using static routing.
```

Comment:
combine the first 2 sentences:
These steps create a redundant GCP IPsec VPN using static routing, used by both route-based Cloud VPN and policy-based Cloud VPN.